src/graphics/program-lib/chunks/reproject.frag

```diff
@@ -135,8 +143,29 @@ vec4 sampleCubemap(vec2 sph) {
     return sampleCubemap(fromSpherical(sph));
 }
 
-vec3 getDirectionEquirect() {
-    return fromSpherical((vec2(vUv0.x, 1.0 - vUv0.y) * 2.0 - 1.0) * vec2(PI, PI * 0.5));
+vec4 sampleEquirect(vec2 sph, float mipLevel) {
+    vec2 uv = sph / vec2(PI * 2.0, PI) + 0.5;
+#if SUPPORTS_TEXLOD == 1
```

Comment:
do you need '== 1' part? should just be #if defined(SUPPORTS_TEXLOD)

Reply:
or similar to this: `#ifdef GL2`

Reply:
I always define SUPPORTS_TEXLOD, but set it to 0 or 1 depending on the device support.

I like defining all the defines (because it acts to document all the options), but it does make the use of it a little less readable.

Reply:
It's not consistent with the rest of the engine .. I'd prefer consistency here.

Reply:
done, thanks.